enterprise/app/code/code.tsx

```diff
@@ -626,6 +639,7 @@ export default class CodeComponent extends React.Component<Props, State> {
       .then(() => {
         window.open(this.state.prLink, "_blank");
         this.handleClearPRClicked();
+        this.handleUpdateCommitSha(() => {});
```

Comment:
```suggestion
        this.handleUpdateCommitSha();
```

Reply:
There's a comment on `handleUpdateCommitSha` that says:
```
  // If a callback is set, alert messages will not be shown.
```

And I don't want to show a `You're already up to date!` toast in this case.